pkg/controllers/uiplugin/compatibility_matrix.go

```diff
@@ -81,6 +81,15 @@ var compatibilityMatrix = []CompatibilityEntry{
 			"dev-alerts",
 		},
 	},
+	{
+		PluginType:        uiv1alpha1.TypeMonitoring,
+		MinClusterVersion: "v4.14",
+		MaxClusterVersion: "",
+		ImageKey:          "ui-monitoring",
+		Features: []string{
+			"acm-alerting",
```

Comment:
can we detect we are running in an ACM hub cluster?

Reply:
ACM detection has been added in the most recent changes. As per the conversation [here](https://redhat-internal.slack.com/archives/C063J3F88MV/p1729256903923739?thread_ts=1729058727.278719&cid=C063J3F88MV), we will be reconciling the uiplugin based on ACM version, just as the cluster version is also taken into account.